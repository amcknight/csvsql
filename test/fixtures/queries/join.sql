SELECT P.first_name, P.last_name
FROM test/fixtures/tables/people as P
JOIN test/fixtures/tables/cool as C
  ON P.first_name = C.name
WHERE C.isCool
