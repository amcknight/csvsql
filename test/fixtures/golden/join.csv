first_name, last_name
Cayley, Carlson
July, McKnight
